src/App.css

```diff
@@ -11,7 +11,7 @@
   font-weight: bold;
 }
 
-.route-links {
+nav {
   display: flex;
   flex-direction: column;
   align-items: center;
```

Comment:
You can choose to have the CSS in a separate file or use something like emotion so you can have your CSS inline and pass props to components. It's pretty exciting; if you'd like to know more https://emotion.sh/docs/introduction

Reply:
Oooh thanks for the heads up about emotion!